soccer/gameplay/plays/offense/basic_122.py

```diff
@@ -11,6 +11,11 @@
 
 class Basic122(standard_play.StandardPlay):
 
+    situationList = [
```

Comment:
So can super score see this?

Reply:
I would prefer an `is_valid(situation)` function instead of just a public list.

Reply:
I can make an is_valid but would still prefer making an "private" _situationList to pass to situation analysis and to make it so that all you have to do is override that variable to make everything work at a basic level and allow you to use the super score function as is. is_valid would basically just be a contains call though if I'm understanding right.

Reply:
That sounds like the correct fix.